OpenTrons/opentrons-api
# feat(shared-data, labware-library): add deck riser definition and new Lid category to LL

re AUTH-869, AUTH-892, PLAT-533, and AUTH-887



# Overview



Adding a new deck riser definition with its image to shared data so it can be displayed in the Labware Library, and updating the TC lid definition to include a stacking offset for the deck riser. Also, adding a new 'Lid' category to the Labware Library for the TC auto-sealing lids.

## Test Plan and Hands on Testing
- Make sure the Opentrons Flex Deck Riser shows up in the Labware Library's "Adapter" category
- Ensure that the “Lid” category appears on the left navigation of the Labware Library site
- When the “Lid” category is selected, the Opentrons Tough PCR Auto-Sealing Lid appears in the search results

For testing deck riser definition on a Flex:
- Run a protocol that have the gripper move a stack of TC lids from and to the deck riser. Example protocol I used: 
[deck_riser_with_tc_lid_test1.zip](https://github.com/user-attachments/files/17463215/deck_riser_with_tc_lid_test1.zip)




## Changelog


- Added the opentrons_flex_deck_riser definition to shared-data after tested it with TC lids on the Flex
- Updated the opentrons_tough_pcr_auto_sealing_lid definition and added its image to LL
- Updated files to include a 'Lid' category in LL

## Review requests



## Risk assessment
